classes/Commands.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+""" API module holding all the registered commands. Also contains base command classed. """
+# TODO: to whoever has an idea on how imports work, please import the main module here. Then update all the objects with
+# the ChatRoomProtocol class. Thanks!
+
+
+class Command:
+    def __init__(self, name: str, permissionLevel: int, action: callable(object, str)):
```

Comment:
`callable` is a built-in that tests whether an object can be called as a function. For type hints, perhaps you mean `typing.Callable[object, str]` here?